projects/Mallard/src/constants/settings/faq.json

```diff
@@ -1 +1,3 @@
re you choose which issue to read) press the refresh button in the blue bar at the top. If there is no response, check your internet connection by going to Safari and browsing to two or three pages. If you are sure you have a connection, and you also have battery power, close down the app and re-open.</p> <p><strong>How do I access issues that are older than 7 days?</strong><br>The app allows access to a rolling seven days of editions by default. To change the number of issues you can access, go to Settings &gt;&gt; Advanced settings &gt;&gt; Forget issues older than and from there choose the option that best suits you.</p> <p><strong>How do I share content?</strong><br>From an article page, tap the share icon underneath the issue name in the top right hand side of the screen. This will allow you to share via email, Twitter, Instapaper or Facebook. Please note that you can only share articles that are live on our website.</p> <p><strong>How can I find related content in the same issue?</strong><br>When there is related content it will be highlighted in the column to the right hand side of the article text. There are links to related stories within the issue and also further reading on our website.</p> <p><strong>How do I access crosswords?</strong><br>Crosswords are available as a menu item, or by swiping through the sections of the app. Please note: you must be connected to the internet to retrieve crosswords, but once they are downloaded you can play offline.</p> <p><strong>Can I use the app when I’m abroad?</strong><br>We advise checking with your operator prior to using the app abroad as most monthly plans charge extra for data roaming while overseas. As this can often be expensive, either sync using Wi-Fi or contact your service provider for details of how to purchase cut-price roaming tariffs.</p> <p><strong>How do I change the font size?</strong><br>The current version of the app does not have an option to change the font size.</p> <p><strong>Why can’t I watch video?</strong><br>You need an internet connection to watch video. Please check your connection and try again.</p> <p><strong>Why do some issues in the issue picker have an “Update?” button? What does this mean?</strong><br>Issues are updated to pick up stories from later editions of that day’s paper or to amend the issue for legal reasons. The update button will re-download the latest version of that issue.</p> <h2>Feedback</h2> <p><strong>How do I give you feedback/ideas on how the app can be improved?</strong><br>If you have trouble subscribing please email <a href=\"mailto:tablet@theguardian.com\">tablet@theguardian.com</a>. If you are having problems with the app or have any suggestions on how we can improve please email <a href=\"mailto:apps.feedback@theguardian.com\">apps.feedback@theguardian.com</a>.</p> <p><strong>Couldn’t find what you were looking for?</strong></p> <p><strong><a href=\"https://www.theguardian.com/help\">Click here to access our Help site</a></strong></p>"}
```

Comment:
Linting changed this. Only change here was the email. Im concerned that this might be autogenerated?